cypress/e2e/home.cy.js

```diff
@@ -0,0 +1,69 @@
+describe('Viewing Home page', () => {
+  // declare variables that can be used to change how the response is intercepted.
+  let loading
+  let error
+  let featuredServices
+
+  beforeEach(() => {
+    // Intercept the response from the endpoint to view all requests
+    cy.customApiIntercept({
+      action: 'GET',
+      alias: 'useAllWares',
+      requestURL: `/providers/${Cypress.env('NEXT_PUBLIC_PROVIDER_ID')}/wares.json`,
+      data: featuredServices,
+      defaultFixture: 'services/wares.json',
+      emptyFixture: 'services/no-wares.json',
+      loading,
+      error
+    })
+    cy.visit('/')
+  })
+
+
+  context('featured services list is loading', () => {
+    before(() => {
+      loading = true
+    })
+    it('should show 3 placeholder cards loading', () => {
+      cy.get('p.placeholder-glow').should('be.visible').then(() => {
+        cy.log('Loading text displays correctly.')
+      })
+    })
+  })
+
+  context('error while making a request to the api', () => {
+    before(() => {
+      featuredServices = undefined
```

Comment:
```suggestion
```

Reply:
you should be able to delete this line as well since it is already undefined after you declared it on line 5. But double check to make sure the tests still pass before accepting this one